recipes/level-zero/all/patches/1.17.28/001-patch-install-rules.patch

```diff
@@ -0,0 +1,116 @@
+diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
+index 5ef252c..c9d05dd 100644
+--- a/source/CMakeLists.txt
++++ b/source/CMakeLists.txt
+@@ -49,21 +49,10 @@ endif()
+ target_link_libraries(${TARGET_LOADER_NAME} utils)
+ 
+ install(TARGETS ze_loader
+-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+-    LIBRARY
+-        DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-        COMPONENT level-zero
+-        NAMELINK_SKIP
+-)
+-
+-install(TARGETS ze_loader
+-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+-    LIBRARY
+-        DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-        COMPONENT level-zero-devel
+-        NAMELINK_ONLY
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT level-zero-devel
++    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT level-zero
++    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT level-zero
++    NAMELINK_COMPONENT level-zero-devel
```

Comment:
Installation rules update is merged https://github.com/oneapi-src/level-zero/pull/180 and port of official releases since [v1.17.39](https://github.com/oneapi-src/level-zero/releases/tag/v1.17.39) version

Reply:
Updated